build.gradle

```diff
@@ -100,9 +100,6 @@ node {
 //──────────────────────────────────────────────────────────────────────────────
 sourceSets {
 	main {
-		java {
```

Comment:
I probably copied this section from somewhere, I have no idea what it does, so as long as the app works, it's A OK to me.